content/docs/start/data-and-model-versioning.md

```diff
@@ -5,15 +5,15 @@ regular Git workflow for datasets and ML models, without storing large files in
 Git.'
 ---
 
-# Get Started: Data Versioning
+# Data Versioning
 
 How cool would it be to make Git handle arbitrary large files and directories
 with the same performance as with small code files? Imagine doing a `git clone`
 and seeing data files and machine learning models in the workspace. Or switching
 to a different version of a 100Gb file in less than a second with a
 `git checkout`.
 
-The foundation of DVC consists of a few commands that you can run along with
+The foundation of DVC consists of a few commands which you can run along with
```

Comment:
Restrictive clause (keep "that"). A good rule of thumb I use: if it should have a comma before the clause, then use "which" (prob not the case here).